scrapers/AVJiali.yml

```diff
@@ -0,0 +1,138 @@
+name: "AV Jiali"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - avjiali.com
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - avjiali.com
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h5/a
+      Date:
+        selector: //div[@class="video-date"]/text()
+        postProcess:
+          - replace:
+              - regex: (\d+)(st|nd|rd|th)
+                with: "$1"
+          - parseDate: January 02, 2006
+      Performers:
+        Name: //div[@class="cat"][1]/a[starts-with(@href, 'https://avjiali.com/model')]
+        URL: //div[@class="cat"][1]/a/@href
+      Tags:
+        Name: //div[@class="cat"][2]/a
+      Details:
+        selector: //h1[@class="none"]
```

Comment:
This is just grabbing the title again, I'd suggest the following to grab the full details
```
Details:
  selector: //p[span[@class="readmore"]]
  postProcess:
    - replace:
        - regex: ... Read More
          with:
```

Reply:
Was just working on this.  The description is in 3 different tags, so I'm having trouble with concatenating it all together.

Reply:
I was looking at the same thing (tried with the `string()` xpath function) but I realized if we just grab it from the `<p>` and remove the `... Read More` bit it works great, did you try the selector I suggested?

Reply:
Thanks, worked great.  I also fixed an issue with the Details: under the performer scraper copied from the tenshigao.yml.